lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -228,7 +228,7 @@ pub trait MetachainRPC {
         &self,
         block_count: U256,
         newest_block: BlockNumber,
-        reward_percentile: Vec<usize>,
+        reward_percentile: Vec<i64>,
```

Comment:
This is the correct behavior with geth since we should allow parsing negative percentile, but the RPC when validating the reward percentile inputs will throw an error log (instead of a parse error). It is handled [here](https://github.com/DeFiCh/ain/pull/2788/files#diff-de20deb2e050968de23b525d258e086272ab361906f12a5526b0b95dd6b160d7R355-R361). Also has a min percentile test inside feature_evm_rpc_fee.

Reply:
Gotcha, a bit confused why it couldn't fit in a u8